aio/aio-proxy/aio_proxy/search/queries/text.py

```diff
@@ -0,0 +1,236 @@
+def build_text_query(terms: str):
+    text_query = {
+        "bool": {
+            "should": [
+                {
+                    "function_score": {
+                        "query": {
+                            "match_phrase": {
+                                "identifiant_association_unite_legale": {
+                                    "query": terms,
+                                    "boost": 50,
+                                    "_name": "exact match identifiant association",
+                                }
+                            }
+                        },
+                        "field_value_factor": {
```

Comment:
this code is used a lot. Maybe use a variable ?

```
etablissement_ouvert_multiplier = {
                            "field": "nombre_etablissements_ouverts",
                            "factor": 1,
                            "modifier": "log2p",
                            "missing": 1,
                        }
```

And then everywhere you need it 

```
"field_value_factor": etablissement_ouvert_multiplier
```